core/src/main/scripts/importer/validateData.py

```diff
@@ -1179,18 +1192,56 @@ def checkNotBlank(self, value, data):
         if value is None or value.strip() == '':
             return False
         return True
-    
+
     def checkSwissProt(self, value, data):
-        """Test whether SWISSPROT string is blank and give warning if blank."""
+        """Validate the accession in the SWISSPROT column."""
         if value is None or value.strip() == '':
             self.logger.warning(
                 'Missing value in SWISSPROT column; this column is '
                 'recommended to make sure that the Uniprot canonical isoform '
                 'is used when drawing Pfam domains in the mutations view',
                 extra={'line_number': self.line_number,
-                       'cause':'blank value in SWISSPROT column'})
-            
-        # it is just a warning, so we can return True always:
+                       'cause':'<blank>'})
+            # no value to test, return without error
+            return True
+        if self.meta_dict.get('swissprot_identifier', 'name') == 'accession':
+            if not re.match(
+                    # regex from http://www.uniprot.org/help/accession_numbers
+                    r'^([OPQ][0-9][A-Z0-9]{3}[0-9]|'
+                    r'[A-NR-Z][0-9]([A-Z][A-Z0-9]{2}[0-9]){1,2})$',
+                     value):
+                # return this as an error
+                self.extra = 'SWISSPROT value is not a UniprotKB accession'
+                self.extra_exists = True
+                return False
+            # test whether the accession is known to the portal, if available
+            if (self.portal.uniprotkb_entry_map is not None and
+                    value not in self.portal.uniprotkb_entry_map):
+                self.logger.warning(
+                    'This UniprotKB/Swiss-Prot accession is not known to the '
+                    'portal, the portal will attempt to display the protein '
+                    'based on the gene',
```

Comment:
strictly speaking, this is not the case anymore. Since some fixes ago, the portal actually looks up the accession directly from the `pfam_graphics` table (see `PfamSequenceServlet.java`), which is not the table that is used to build up the `uniprotkb_entry_map` here. 

A webservice that selects the list of uniprot accessions from `pfam_graphics` table would suffice for the accession scenario.